jenkins/install-deps.sh

```diff
@@ -1,8 +1,41 @@
 #!/bin/bash -xue
+function install_deb {
+    sudo aptitude install -y python-dev libffi-dev python-pip
+}
 
-sudo add-apt-repository --yes ppa:fkrull/deadsnakes
-sudo apt-get update
-sudo apt-get install --yes python2.6 python2.6-dev
+function install_centos {
+    sudo yum -y install python-devel libffi-devel
+    # pip is not in the standard repo
+    sudo yum -y install epel-release
+    sudo yum -y install python-pip
 
-sudo aptitude install -y python-dev libffi-dev python-pip
-sudo pip install tox
+    # This gets installed automatically in debian because recommended packages
+    # Doing that manually in centos.
+    sudo yum -y install gcc
+
+}
+
+
+function is_centos {
+    [[ -f /etc/centos-release ]]
+}
+
+function is_deb {
+    [[ -f /etc/debian_version ]]
+}
+
+function install {
+    if is_deb;
+        then
```

Comment:
I don't want to be picky, but the indentation Emmanuel style is a bit weird.... :)

I'd propose we use : 
```
if is_deb; then
  install_deb
```
And also, no need to end your lines with a semi colon ';'.